dm/loader/lightning.go

```diff
@@ -150,7 +180,8 @@ func (l *LightningLoader) restore(ctx context.Context) error {
 		}
 		cfg.Routes = l.cfg.RouteRules
 		cfg.Checkpoint.Driver = lcfg.CheckpointDriverMySQL
-		cfg.Checkpoint.Schema = config.TiDBLightningCheckpointPrefix + dbutil.ColumnName(l.workerName)
+		cfg.Checkpoint.Schema = config.TiDBLightningCheckpointPrefix + dbutil.TableName(l.workerName, l.cfg.Name)
```

Comment:
I'm not sure why we need `dbutil.TableName`, because in `NewMySQLCheckpointsDB` of lightning it will do escaping for backquotes 🤔 

can we only use `config.TiDBLightningCheckpointPrefix + l.workerName + l.cfg.Name` as database name?

Reply:
It seems that can not pass test